libraries/AP_CheckFirmware/AP_CheckFirmware.h

```diff
@@ -51,16 +54,22 @@ struct app_descriptor {
     // total size of firmware image in bytes
     uint32_t image_size = 0;
     uint32_t git_hash = 0;
+
+    // firmware signature
```

Comment:
this addition to the app_descriptor should be conditional on AP_SIGNED_FIRMWARE_ENABLED